### DISCUSSION TITLE
Install fails on OSX 17 apple m1 chip

### DISCUSSION THREAD

**Discussion by MatthiasWeisz** [WeAreAthlon/frontend-webpack-boilerplate#198] [Q&A] [answered]
Cant npm install on macbook m1 osx 17.
node v14
npm v9

Errors:
command failed
npm ERR! command sh -c node lib/install.js
npm ERR! compiling from source
npm ERR! spawn Unknown system error -86
npm ERR! optipng pre-build test failed
npm ERR! Error: Command failed: /bin/sh -c make install
npm ERR! pngrtran.c:99:1: warning: unused function 'png_rtran_ok' [-Wunused-function]
npm ERR! png_rtran_ok(png_structrp png_ptr, int need_IHDR)
npm ERR! ^
npm ERR! 1 warning generated.
npm ERR! ld: warning: option -s is obsolete and being ignored
npm ERR! Undefined symbols for architecture arm64:
npm ERR!   "_png_init_filter_functions_neon", referenced from:
npm ERR!       _png_read_filter_row in libpng.a(pngrutil.o)
npm ERR! ld: symbol(s) not found for architecture arm64
npm ERR! clang: error: linker command failed with exit code 1 (use -v to see invocation)
npm ERR! make[1]: *** [optipng] Error 1
npm ERR! make: *** [install] Error 2

**Comment by pnikolov** [comment, marked as answer]
@MatthiasWeisz node versions prior to 16 (`<= 16`) does not support natively the arm64 (Apple M1 chip). You should either upgrade to node >= 16 or if you are stuck on v14 then run it via the emulator (emulated terminal also) - you can follow this article: https://devzilla.io/using-nodejs-14-with-mac-silicon-m1

**Comment by devshack** [comment]
It works flawlessly with the latest LTS Version of node 18.8.1 ... just hop over and download it ;)
